errors_validation.go

```diff
@@ -0,0 +1,47 @@
+package jsonschema
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type validationErrorContext interface {
+	addContext(instancePtr, schemaPtr string)
+	finishInstanceContext()
+}
+
+// ValidationRequiredError is used when a
+type ValidationRequiredError struct {
```

Comment:
name ends with 'Error' but does not implement error interface

Reply:
True, `ValidationErrorRequired` or `ValidationContextRequired` is probably a better naming!